lib/todo/console_io_handler.ex

```diff
@@ -0,0 +1,9 @@
+defmodule ToDo.ConsoleIOHandler do
```

Comment:
`ConsoleIOHandler` sounds odd from a user's point of view. When I say user in this case, I mean another developer that might want to use your code. If I was looking for the thing that prints on the screen, `Display`, `ConsoleDisplay` and names like that are more close to what I would look for.